### DISCUSSION TITLE
水系所支持的牌堆格式有几种？

### DISCUSSION THREAD

**Discussion by kenichiLyon** [HydroRoll-Team/support#10] [Q&A] [answered]
随便问问，问题如标题TT

**Comment by HsiangNianian** [comment, marked as answer]
1. json
2. xlsx
3. csv
4. toml
5. yaml(yml)

标准的应该是用 `toml` 但是为了兼容其他的所以才有了 `json` 与 `xlsx` 和 `yaml` 但是既然都有了 `xlsx` 了那就一并兼容一下 `csv`。

**Comment by HsiangNianian** [comment]
## 2023-10-29

### BREAKING CHANGES

* **PublicDeck:** 若加载了`xlsx`格式的牌堆将会给出警告。
